.github/workflows/lint-format.yml

```diff
@@ -12,15 +12,15 @@ jobs:
         run: git config --global core.autocrlf false
 
       - name: Checkout Repo
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
           ref: ${{ github.event.pull_request.head.sha }}
       - name: Set up pnpm
         uses: pnpm/action-setup@v4.0.0
         with:
           version: 8.6.9
-      - name: Set up node v16
-        uses: actions/setup-node@v3
+      - name: Set up node
+        uses: actions/setup-node@v4
         with:
           node-version: 16
```

Comment:
Should this be updated?

Reply:
[d594a2d](https://github.com/evidence-dev/evidence/pull/2235/commits/d594a2d896613ae9afe1cf653648ec7dedbe4dfc)